ui/components/app/assets/asset-list/asset-list.tsx

```diff
@@ -103,19 +105,32 @@ const AssetList = ({ onClickAsset, showTokensLinks }: AssetListProps) => {
   // for EVM assets
   const shouldShowTokensLinks = showTokensLinks ?? isEvm;
 
+  const detectedTokensMultichain = useSelector(
+    getAllDetectedTokensForSelectedAddress,
+  );
+
+  const totalTokens =
```

Comment:
Can this be calculated with the result of my selector?  https://github.com/MetaMask/metamask-extension/pull/28502

Reply:
yeah I think so. I can integrate this soon